services/web/client/source/class/osparc/utils/Utils.js

```diff
@@ -199,22 +199,54 @@ qx.Class.define("osparc.utils.Utils", {
     },
 
     downloadLink: function(url, fileName) {
-      let xhr = new XMLHttpRequest();
-      xhr.open("GET", url, true);
-      xhr.responseType = "blob";
-      xhr.onload = () => {
-        console.log("onload", xhr);
-        if (xhr.status == 200) {
-          let blob = new Blob([xhr.response]);
-          let urlBlob = window.URL.createObjectURL(blob);
-          let downloadAnchorNode = document.createElement("a");
-          downloadAnchorNode.setAttribute("href", urlBlob);
-          downloadAnchorNode.setAttribute("download", fileName);
-          downloadAnchorNode.click();
-          downloadAnchorNode.remove();
+      return new Promise((resolve, reject) => {
+        let xhr = new XMLHttpRequest();
+        xhr.open("GET", url, true);
```

Comment:
as explained above, these custom methods should use the POST verb as they are the most flexible.
and also because exporting does actually create an archive it does not just get it out.